.deploystack/test.yaml

```diff
@@ -16,9 +16,6 @@
 # in the testing rig
 
 steps:
-   - name: 'gcr.io/cloudshell-images/cloudshell:latest'
-     entrypoint: bash
-     args: [ '.deploystack/clean' ]
```

Comment:
**Thought:**
I wonder if it makes more sense to have these tests live in a separate DeployStack-owned repository.
That `git` repo would include all similar tests for DeployStack-enabled applications.
The pros would be:
1. I (and other sample app owners) won't be a bottleneck to your team's changes to these tests.
2. I assume there's some common code between each of the test across the apps. Modularizing and reusing that code would be easier (since all the code would live in the same repo).

This is just an idea. No action needed. :)

Reply:
I've done that with the core functionality. But I've never figured out a good solution for sharing bash code. Any ideas you have would be super welcome by me.